### PR TITLE
meetup-2023-10-19: add flyer, campus-map and extend announcement

### PR DESCRIPTION
**Pull request by msanft**
Add the meetup flyer, extend the announcement text as per the information excluded on the flyer, and add a campus map with instructions on how to reach the room.

### CODE REVIEW

**Comment by katexochen** [comment]
@msanft could you add the flyer source?

File: meetups/2023-10-19/announcement.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+
+### Wo?
+
+Ruhr-Universität Bochum, Raum MC 1.30 / MC 1.31. Eine Anfahrtsbeschreibung findet ihr auf der [Webseite der Uni](https://www.ruhr-uni-bochum.de/de/anreise-zur-ruhr-universitaet). Das Gebäude MC ist grün im beigefügten Campusplan eingezeichnet. Die Räume MC 1.30 / MC 1.31 finden sich sehr zentral im ersten Stock. Kontaktiert uns gerne für einen detaillierten Lageplan zur rollstuhlgerechten Anreise.
+
+### Wer?
+
+Absolut jede*r mit Interesse an Nix(OS). Ihr müsst keine Vorerfahrung mit Nix mitbringen.
+
+## FAQ
```
Comment (katexochen) [inline meetups/2023-10-19/announcement.md:19]: Imo now to long. We should add an initial section without headline and only give the most important info.

File: meetups/2023-10-19/announcement.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+
+19.10.2023, ca. 18:00-22:00
+
+### Wo?
+
+Ruhr-Universität Bochum, Raum MC 1.30 / MC 1.31. Eine Anfahrtsbeschreibung findet ihr auf der [Webseite der Uni](https://www.ruhr-uni-bochum.de/de/anreise-zur-ruhr-universitaet). Das Gebäude MC ist grün im beigefügten Campusplan eingezeichnet. Die Räume MC 1.30 / MC 1.31 finden sich sehr zentral im ersten Stock. Kontaktiert uns gerne für einen detaillierten Lageplan zur rollstuhlgerechten Anreise.
+
+### Wer?
+
+Absolut jede*r mit Interesse an Nix(OS). Ihr müsst keine Vorerfahrung mit Nix mitbringen.
```
Comment (katexochen) [inline meetups/2023-10-19/announcement.md:17]: Second sentence could be added to "Was solltest du mitbringen?" section

File: meetups/2023-10-19/announcement.md (hunk 1 above)
Comment (msanft) [reply meetups/2023-10-19/announcement.md:19]: So replace "Infos" with the title of the announcement and remove everything before that?

File: meetups/2023-10-19/announcement.md (hunk 2 above)
Comment (msanft) [reply meetups/2023-10-19/announcement.md:17]: IMO this doesn't fit that well as the section below rather mentions material things that should be brought and doesn't mention the "soft skills"

File: meetups/2023-10-19/announcement.md (hunk 1 above)
Comment (katexochen) [reply meetups/2023-10-19/announcement.md:19]: Time and date is on the flyer already. Room should be on the flyer, too, just shorten Ruhr-Uni... to "RUB". Where should the campusplan be published? I don't think this doable or nice as PDF. 

Iterating on this PR is super slow, lets merge everything and return announcement discussion to issue.

**Pull request merged**